atomic_reactor/plugins/post_rpmqa.py

```diff
@@ -43,6 +44,9 @@ def __init__(self, tasker, workflow, image_id, ignore_autogenerated_gpg_keys=Tru
         self.ignore_autogenerated_gpg_keys = ignore_autogenerated_gpg_keys
 
     def run(self):
+        if base_image_is_scratch(self.workflow.builder.base_image.to_str()):
+            self.log.debug("image has scratch base, do not perform RPM check")
+            return
```

Comment:
Yea you're right. Cannot check this atm but I guess we would want to implement a non-fatal exit and warning log when rpm itself is not installed.